File: android/app/src/main/kotlin/com/example/hotel_reception/procesor/dto/enums.kt
package com.example.hotel_reception.procesor.dto

enum class ServiceDto {
    normal, medium, expensive,
}

enum class RoomTypeDto {
    econom,
    suite,
    president,
}

enum class WorldSpeed(val time: Long) {
    normal(3000L), fast(2000L), faster(1000L),
}

File: ios/Runner/Processor/dto/enums.swift
//
//  enums.swift
//  Runner
//
//  Created by Serhii Rakovskyi on 17.08.2022.
//

import Foundation

enum ServiceDto: Decodable {
    case normal
    case medium
    case expensive
}

enum RoomTypeDto: Decodable {
    case econom, suite, president
}
